ctms/acoustic_service.py

```diff
@@ -267,21 +267,27 @@ def _waitlist_converter(self, acoustic_main_table, contact, main_fields):
         If the field `{name}_waitlist_{field}` is not present in the `main_fields`
         list, then it is ignored.
         See `bin/acoustic_fields.py` to manage them (eg. add ``vpn_waitlist_source``).
-
-        Note: In the future, a dedicated relation/table for waitlists can be considered.
         """
         waitlists_by_name = {wl.name: wl for wl in contact.waitlists}
+        # Publish all potential fields (`waitlist.<field>` or `waitlist.fields[<field>]`)
         for acoustic_field_name in main_fields:
-            if "_waitlist_" not in acoustic_field_name:
-                continue
-            name, _, field = acoustic_field_name.split("_")
-            value = None
-            if name in waitlists_by_name:
-                waitlist = waitlists_by_name[name]
-                value = getattr(waitlist, field, waitlist.fields.get(field, None))
-            acoustic_main_table[
-                acoustic_field_name
-            ] = self.transform_field_for_acoustic(value)
+            if acoustic_field_name.startswith("sub_") and acoustic_field_name.endswith(
+                "_waitlist"
+            ):
+                _, name, _ = acoustic_field_name.split("_")
+                acoustic_main_table[acoustic_field_name] = (
+                    "1" if name in waitlists_by_name else None
```

Comment:
I'm not sure it's possible that `name=super_product` since the `.split` to extract `name` is using the "_" character:
```
_, name, _ = acoustic_field_name.split("_")
```

----

In terms of the `waitlist.name`
if it was `super-product` then let's say the 
`acoustic_field_name` is `sub_super-product_waitlist`--however, that's not possible since I believe that the "-" is reserved in acoustic? 

So perhaps if we roll back to data-input...we may need some input validation... and "test" that the acoustic_main table doesn't allow for this type of ["main_field"](https://github.com/mozilla-it/ctms-api/blob/3dea45973c76bf6f718b954a4956d62ec22e48a6/ctms/sync.py#L137-L141)

https://github.com/mozilla-it/ctms-api/blob/3dea45973c76bf6f718b954a4956d62ec22e48a6/ctms/bin/acoustic_fields.py#L49-L61

Perhaps a model validation constraint on AcousticField.field?
https://github.com/mozilla-it/ctms-api/blob/3dea45973c76bf6f718b954a4956d62ec22e48a6/ctms/models.py#L212-L220